lms/envs/common.py

```diff
@@ -1917,6 +1922,15 @@
 CERT_NAME_SHORT = "Certificate"
 CERT_NAME_LONG = "Certificate of Achievement"
 
+#################### Badgr OpenBadges generation #######################
+BADGR_API_TOKEN = None
+BADGR_BASE_URL = "http://localhost:8005"
+BADGR_ISSUER_SLUG = "example-issuer"
+# A dictionary of course modes to file paths. Example:
+# {'honor': '/path/to/badge_image.png'}
+# Badge images must be square and in PNG or SVG format.
+BADGR_IMAGE_SOURCES = {}
```

Comment:
@antoviaque I don't believe Badgr-Server has a method for updating a badge image. It's set in stone, at least in theory. However, I believe if one were particularly enterprising they could replace the image file in Badgr-server manually, and new badges would be created that way.